package.json

```diff
@@ -2,15 +2,15 @@
   "name": "madison",
   "description": "A dirt simple utility for working with US state names and abbreviations.",
   "author": "Mike Ball <mikedball@gmail.com>",
-  "version": "0.0.9",
+  "version": "1.0.0",
```

Comment:
I'm curious to hear a bit more about your logic in thinking this makes semantic versioning easier in the long run.